test/Microsoft.SqlTools.ServiceLayer.UnitTests/EditData/ServiceIntegrationTests.cs

```diff
@@ -189,7 +189,12 @@ public async Task RevertRowSucceeds()
 
             // If: I ask to revert a row that has a pending edit
             var efv = new EventFlowValidator<EditRevertRowResult>()
-                .AddResultValidation(Assert.NotNull)
+                .AddResultValidation(result =>
+                {
+                    Assert.NotNull(result);
```

Comment:
Nit: use fluent-style assertions to make it easier to read and make it obvious which value is expected vs. actual:

e.g.: `Assert.That(something, Is.Not.True);` and `Assert.That(something, Is.EqualTo(exepectedVal))`